source/structured-streaming.txt

```diff
@@ -160,6 +213,74 @@ more about continuous processing, see the `Spark documentation <https://spark.ap
 
    tabs:
 
+     - id: java-sync
+       content: |
+
+         To use continuous processing with the MongoDB Spark Connector, 
+         add the ``trigger()`` method to the ``writeStream`` property 
+         of the streaming Dataset or DataFrame that you create from 
+         your MongoDB read stream. In your ``trigger()``, use the 
+         ``Trigger.Continuous()`` property.
+         
+         .. note:: 
+         
+            The connector populates its read stream from your MongoDB 
+            deployment's change stream. To populate your change stream, 
+            perform update operations on your database.
+         
+            To learn more about change streams, see 
+            :manual:`Change Streams </changeStreams>` in the MongoDB 
+            manual.
+         
+         Specify read stream configuration settings on your local 
+         SparkSession ``readStream``. You must specify the following 
+         configuration settings to read from MongoDB:
+         
+         .. list-table::
+            :header-rows: 1
+            :stub-columns: 1
+            :widths: 10 40
+         
+            * - Setting
+              - Description
+         
+            * - ``readStream.format()``
+              - The format to use for read stream data. Use ``mongodb``.
+         
+            * - ``writeStream.trigger()``
+              - Enables continuous processing for your read stream. 
```

Comment:
Issue: This seems like a time setting for how frequently it should poll for changes (if this is the correct documentation): https://spark.apache.org/docs/latest/api/java/org/apache/spark/sql/streaming/Trigger.html

Suggestion: Use a description that indicates that it sets when it should read changes, perhaps similar to the API documentation description.